numpy/core/tests/test_item_selection.py

```diff
@@ -68,6 +68,21 @@ def test_unicode_mode(self):
         k = b'\xc3\xa4'.decode("UTF8")
         assert_raises(ValueError, d.take, 5, mode=k)
 
+    def test_partition(self):
```

Comment:
should be called `test_empty_partition` to make clear what's special about this test